lib/resolvers/membership_request_mutations/accept_membership_request.js

```diff
@@ -29,9 +31,11 @@ module.exports = async (parent, args, context) => {
 
   //check to see if user is already a member
   org._doc.members.forEach((member) => {
-    if (member._id === user.id) {
+    if (member._id.toString() === user._id.toString()) {
```

Comment:
May I ask why are we converting the two IDs to a string? If they weren't changed to a string, wouldn't they evaluate in the same way?

Reply:
I've faced the string error a lot so I just explicitly converted every id to a string to avoid confusion.

Reply:
would you happen to know why the string error happened quite a bit? Sounds as though, sometimes you're experiencing it and sometimes you aren't

Reply:
Some ids are coming from the database and some of them are coming form the parameters.

Reply:
And it becomes confusing to reason which is which, also I found errors from using .id without the _id or using a whole object instead of just adding the id.